requirements.txt

```diff
@@ -0,0 +1 @@
+pyradios~=0.0.22
```

Comment:
Hmm, I mean, using `~=` here is super correct but I tend to use `==` and then have @dependabot create a pull request to update the requirements version. This then first gets committed to the master branch which every Friday with my approval gets merged into stable so we have a few days where the "beta testers" (who probably don't exist) can notice breakage before it ends up in stable. I don't know what dependabot will do with `~=`